src/components/translator/Translator.tsx

```diff
@@ -272,6 +277,14 @@ const Translator = ({ mode: initialMode }: { mode?: Mode }): React.ReactElement
 
   const onTranslate = React.useCallback(() => window.dispatchEvent(new Event(TranslateEvent)), []);
 
+  const swapLangs = React.useCallback(() => {
+    setSrcText(tgtText);
+  }, [setSrcText, tgtText]);
+
+  const swapLangText = () => {
+    swapLangs();
+  };
```

Comment:
This function doesn't have any purpose now? I think we can just pass `swapLangs` around now. It's also a bit odd that `swapLangs` only has one thing happen inside it. Can we also move the actual language swap there instead of `src/components/translator/LanguageSelector.tsx`?